leap_drops/CallLeap.py

```diff
@@ -0,0 +1,89 @@
+import json
+import os
+import random
+import subprocess
+import time
+
+from dlg.droputils import DROPFile
+from dlg.drop import BarrierAppDROP
+from dlg.meta import dlg_int_param, dlg_float_param, dlg_string_param, \
+    dlg_component, dlg_batch_input, dlg_batch_output, dlg_streaming_input
+
+## Call Leap
+# @brief Call Leap
+# @details A BarrierAppDrop that reads a config file, generates a command line for the LeapAccelerateCLI application, and then executes the application
+# @par EAGLE_START
+# @param gitrepo $(GIT_REPO)
+# @param version $(PROJECT_VERSION)
+# @param category PythonApp
+# @param[in] param/measurementSetFilename/-/String
+#     \~English The file from which the input measurement set should be loaded\n
+#     \~Chinese 要读取的起始频率\n
+#     \~
+# @param[in] param/appclass/leap_nodes.CallLeap.CallLeap/String
+#     \~English The path to the class that implements this app\n
+#     \~Chinese 要读取的起始频率\n
+#     \~
+# @param[in] port/Config
+#     \~English The Config file containing JSON specifying how this instance of LeapAccelerateCLI should be run
+#     \~Chinese 要读取的起始频率\n
+#     \~
+# @param[out] port/Result
+#     \~English The output of the LeapAccelerateCLI application (JSON)
+#     \~Chinese 要读取的起始频率\n
+#     \~
+# @par EAGLE_END
+
+class CallLeap(BarrierAppDROP):
+    """A BarrierAppDrop that reads a config file, generates a command line for the LeapAccelerateCLI application, and then executes the application"""
+    compontent_meta = dlg_component('Call Leap', 'Call Leap.',
+                                    [dlg_batch_input('binary/*', [])],
+                                    [dlg_batch_output('binary/*', [])],
+                                    [dlg_streaming_input('binary/*')])
+
+    # TODO: this measurementSetFilename is not being read by dlg_string_param
+    #       hard-coding it for the moment
+    #measurementSetFilename = dlg_string_param('measurementSetFilename', '')
+    measurementSetFilename = "/Users/james/working/leap-accelerate/testdata/1197638568-32.ms"
+
+    DEBUG = True
```

Comment:
Does this need to be removed?

Reply:
I've removed the commented line (46). But we do need the hard-coded measurementSetFilename in the short-term. We need to fix an issue in EAGLE before this can be done the "right" way.

Reply:
I was mostly trying to point out the DEBUG flag. It sounds like there is a task to update this in future shortly so I'll wait until then before looking it over some more

Reply:
I think this DEBUG feature is useful for rapid iterations during debugging, so we'd like leave it in for the short term.

Also, we'd like the content in this pull request to make it to master, so that it will be visible to the Travis CI step that automatically produces the EAGLE components.

Would you be happy to approve (and merge) this PR? It would help us make progress on YAN-407. Thanks.